src/functionalTest/resources/application.yaml

```diff
@@ -99,6 +102,10 @@ cdam:
     url: http://localhost/cdam
     enabled: false
 
+
+payment:
+  api:
+    url: http://localhost:8583/payment
```

Comment:
Would this need to be different for AAT and so forth? Not 100% on this. I would assume though for when the functional tests run it would refer to the normal resources yaml config

Correct me if I am wrong of course

Reply:
yes you are wright added in the environment variable.